Frends.AmazonSQS.Send/Frends.AmazonSQS.Send/Definitions/Result.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Frends.AmazonSQS.Send.Definitions;
+
+/// <summary>
+/// Result.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// Gets a value indicating whether the Task was executed successfully.
+    /// </summary>
+    /// <example>true</example>
+    public bool Success { get; private set; }
+
+    /// <summary>
+    /// MessageId.
+    /// </summary>
+    /// <example>aae72806-1469-4663-b3d3-5228d163baa6</example>
+    public string MessageId { get; private set; }
+
+    /// <summary>
+    /// HTTP status code of the operation.
```

Comment:
Is 'OK' a HTTP status code? You might need to change the description.

Reply:
It is actually. There's no numeric code to get from the response.